cfgov/v1/forms.py

```diff
@@ -314,3 +317,53 @@ def get_query_strings(self):
             'tags__name__in',        # topics
             'authors__name__in',     # authors
         ]
+
+
+class NewsroomFilterForm(FilterableListForm):
+
+    def get_pages(self, parent, hostname):
+        pages = CFGOVPage.objects.live_shared(hostname).descendant_of(parent)
+        blogs = []
+        try:
+            blog = CFGOVPage.objects.get(slug='blog')
+            blogs = CFGOVPage.objects.live_shared(hostname).descendant_of(blog)
+        except CFGOVPage.DoesNotExist:
+            print 'A blog landing page needs to be made'
```

Comment:
I don't want to prevent the page from being viewable if only one portion of content doesn't exist. Is there a better way to show that it needs to be created without totally erroring out?